### DISCUSSION TITLE
OMD version 5.50 release when?

### DISCUSSION THREAD

**Discussion by BeratGuzel** [ConSol/omd#199] [General]
Hi,

We are looking to upgrade OMD to the latest version, but we want to ensure that Thruk 3.18 is included. 

Since Thruk 3.18 will be part of the next OMD release, does anyone know when that version is expected to be available?

Thank you!

**Comment by sni** [comment]
There is a roadmap here: https://omd.consol.de/docs/omd/#roadmap

**Reply by BeratGuzel** [reply]
thanks you for the reply!

I have an additional question: Would it be possible to manually upgrade Thruk to version 3.18 after installing OMD 5.40?

**Reply by sni** [reply]
You can update to a nightly OMD builds from the testing or stream repository which contain the latest Thruk builds already.

**Reply by systemdlete** [reply]
I see that there is a 5.50 release; installing now.  Is there a way to be emailed or otherwise notified when new releases of OMD drop?

**Reply by lausser** [reply]
You may subscribe to the rss feed here: https://omd.consol.de/blog/
